transformer.go

```diff
@@ -0,0 +1,25 @@
+// Copyright (C) 2017 ScyllaDB
+// Use of this source code is governed by a ALv2-style
+// license that can be found in the LICENSE file.
+
+package gocqlx
+
+import (
+	"reflect"
+
+	"github.com/gocql/gocql"
+)
+
+// bindTransformer is called right before binding a value to a named parameter.
+type bindTransformer func(name string, val interface{}) interface{}
```

Comment:
Should be exported.

Reply:
I think we can leave is as `Transformer`, it looks generic enough to use it with parsing rows if needed.

Reply:
Named it as `Transformer`